bank-balance-exactly-once/src/main/java/com/github/simplesteph/udemy/kafka/streams/BankBalanceExactlyOnceApp.java

```diff
@@ -53,22 +47,21 @@ public static void main(String[] args) {
         initialBalance.put("time", Instant.ofEpochMilli(0L).toString());
 
         KTable<String, JsonNode> bankBalance = bankTransactions
-                .groupByKey(Serdes.String(), jsonSerde)
+                .groupByKey(Serialized.with(Serdes.serdeFrom(String.class),Serdes.serdeFrom(JsonNode.class)))
                 .aggregate(
                         () -> initialBalance,
                         (key, transaction, balance) -> newBalance(transaction, balance),
-                        jsonSerde,
-                        "bank-balance-agg"
+                        Materialized.as("bank-balance-agg")
                 );
 
-        bankBalance.to(Serdes.String(), jsonSerde,"bank-balance-exactly-once");
+        bankBalance.toStream();
 
-        KafkaStreams streams = new KafkaStreams(builder, config);
+        KafkaStreams streams = new KafkaStreams(builder.build(), config);
         streams.cleanUp();
         streams.start();
 
         // print the topology
-        System.out.println(streams.toString());
+        System.out.println(streams.localThreadsMetadata());
```

Comment:
`streams.localThreadMetadata.foreach(System.out.println)` or something like that